src/bp/bpfs.ts

```diff
@@ -99,6 +100,7 @@ class BPFS {
     const useForce = process.argv.includes('--force')
     const dryRun = process.argv.includes('--dry')
     const keepRevisions = process.argv.includes('--keep-revisions')
+    const requestConfig: AxiosRequestConfig = { timeout: ms('20m') }
```

Comment:
This could be moved to the getPushAxiosClient() method instead, we already use provide headers and other parameters for the client there.